kinto/tests/test_views_buckets.py

```diff
@@ -70,6 +70,18 @@ def test_wrong_create_permissions_cannot_be_added_on_buckets(self):
                           headers=self.headers,
                           status=400)
 
+    def test_buckets_can_handle_arbitrary_attributes(self):
+        bucket = MINIMALIST_BUCKET.copy()
+        fingerprint = "5866f245a00bb3a39100d31b2f14d453"
+        bucket['data'] = {'fingerprint': fingerprint}
+        resp = self.app.put_json('/buckets/beers',
```

Comment:
While reading this I found that it makes actually little sense for "beers" to have a "fingerprint". We might want to do another pass on the examples here to use something that actually makes more sense to the reader. This could be done in another issue.